app-emulation/xen-tools/xen-tools-4.16.4_pre1.ebuild

```diff
@@ -385,11 +376,11 @@ src_prepare() {
 
 	# Remove -Werror
 	find . -type f \( -name Makefile -o -name "*.mk" \) \
-		 -exec sed -i \
-		 -e 's/-Werror //g' \
-		 -e '/^CFLAGS *+= -Werror$/d' \
-		 -e 's/, "-Werror"//' \
-		 {} + || die
+		-exec sed -i \
```

Comment:
Unrelated, although potentially desirable, whitespace only change. Please drop it from the commit.

Reply:
Since 4.17 was a copy of 4.16, I wanted to make them equal, it's just the whitespace change that was fixed in 4.17 and not backported to 4.16 before.